release/RELEASE.md

```diff
@@ -70,8 +72,8 @@ our luck with a random new one.
 
 1. Run through the following test plan on one of Linux or MacOS:
 
-   1. Install the new SDK using `curl -sSL https://get.daml.com/ | sh -s X.XX.XX`,
-      where `X.XX.XX` is the new version number.
+   1. Install the new SDK using `curl -sSL https://get.daml.com/ | sh -s $(cat
```

Comment:
Maybe worth pointing out that this only works if your local state is after the release PR has been merged.